src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,99 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classnames from 'classnames';
+import { MovieCard } from '../MovieCard';
+import { getData } from '../../api/movies';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+type Props = {
+  callback: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ callback }) => {
+  const film = {
+    Poster: '',
+    Title: '',
+    Plot: '',
+    imdbID: '',
+  };
+
+  const [title, setValue] = useState('');
+  const [movie, setMovie] = useState(film);
```

Comment:
you also can only use here null, it will be better, because you no need object film

Reply:
Agree, put null as default value

Reply:
Also, if you're not using primitive types in useState hook, you better use generic for it
```suggestion
  const [movie, setMovie] = useState<Movie | null>(null);
```